build/makelib/golang.mk

```diff
@@ -118,36 +118,16 @@ endif
 go.init: $(GLIDE_INSTALL_STAMP)
 	@:
 
-define go.project
-go.build.packages.$(1):
-	@echo === go build $(1) $(PLATFORM)
-	@$(3) $(GO) build -v -i -o $(GO_OUT_DIR)/$(1)$(GO_OUT_EXT) $(4) $(2)
-
-go.build.packages: go.build.packages.$(1)
-
-go.install.packages.$(1):
-	@echo === go install $(1) $(PLATFORM)
-	@$(3) $(GO) install -v $(4) $(2)
-go.install.packages: go.install.packages.$(1)
-endef
-$(foreach p,$(GO_STATIC_PACKAGES),$(eval $(call go.project,$(lastword $(subst /, ,$(p))),$(p),CGO_ENABLED=0,$(GO_STATIC_FLAGS))))
-
-define go.test.project
-go.build.test.packages.$(1):
-	@echo === go build test $(1) $(PLATFORM)
-	@$(3) $(GO) test -v -i -c -o $(GO_TEST_OUTPUT)/$(1)$(GO_OUT_EXT) $(4) $(2)
-
-go.build.test.packages: go.build.test.packages.$(1)
-endef
-$(foreach p,$(GO_TEST_PACKAGES),$(eval $(call go.test.project,$(lastword $(subst /, ,$(p))),$(p),CGO_ENABLED=0,$(GO_STATIC_FLAGS))))
-$(foreach p,$(GO_LONGHAUL_TEST_PACKAGES),$(eval $(call go.test.project,$(lastword $(subst /, ,$(p))),$(p),CGO_ENABLED=0,$(GO_STATIC_FLAGS))))
-
-
 .PHONY: go.build
-go.build: go.build.packages go.build.test.packages
+go.build:
+	@echo === go build $(PLATFORM)
+	$(foreach p,$(GO_STATIC_PACKAGES),@CGO_ENABLED=0 $(GO) build -v -i -o $(GO_OUT_DIR)/$(lastword $(subst /, ,$(p)))$(GO_OUT_EXT) $(GO_STATIC_FLAGS) $(p)${\n})
```

Comment:
what functional purpose does the addition of the `\n` newline do?

Reply:
a recipe line must end in a newline, and I want generate the newline here.